clients/hadoopfs/src/main/java/io/lakefs/LakeFSFileSystem.java

```diff
@@ -418,32 +381,33 @@ private boolean renameObject(LakeFSFileStatus srcStatus, Path dst) throws IOExce
             objects.stageObject(dstObjectLoc.getRepository(), dstObjectLoc.getRef(), dstObjectLoc.getPath(),
                     creationReq);
         } catch (ApiException e) {
-            throw translateException("renameObject: src:" + srcStatus.getPath() +", dst: " + dst + ", failed to stage object", e);
+            throw translateException("renameObject: src:" + srcStatus.getPath() + ", dst: " + dst + ", failed to stage object", e);
         }
 
         // delete src path
         try {
             objects.deleteObject(srcObjectLoc.getRepository(), srcObjectLoc.getRef(), srcObjectLoc.getPath());
         } catch (ApiException e) {
-            throw translateException("renameObject: src:" + srcStatus.getPath() +", dst: " + dst +
+            throw translateException("renameObject: src:" + srcStatus.getPath() + ", dst: " + dst +
                     ", failed to delete src", e);
         }
         return true;
     }
 
     /**
      * Translate {@link ApiException} to an {@link IOException}.
+     *
      * @param msg the message describing the exception
-     * @param e the exception to translate
+     * @param e   the exception to translate
      * @return an IOException that corresponds to the translated API exception
      */
     private IOException translateException(String msg, ApiException e) {
         int code = e.getCode();
         switch (code) {
             case HttpStatus.SC_NOT_FOUND:
-                return (IOException)new FileNotFoundException(msg).initCause(e);
+                return (IOException) new FileNotFoundException(msg).initCause(e);
```

Comment:
Why do we even need the casts?  Just to be sure it's not some random RuntimeException we're throwing?

Reply:
Will check - I think this code was updated because I triggered "code format".

Reply:
The method `initCause` returns `Throwable` and the original exception type. fixing.